test/rebar3_ex_doc_SUITE.erl

```diff
@@ -325,11 +325,11 @@ mermaid_before_before_closing_body_tag(Config) ->
         name => AppNameStr,
         config =>
             {ex_doc, [
-                {source_url, <<"https://github.com/eh/eh">>},
-                {extras, [<<"README.md">>, <<"LICENSE">>]},
-                {main, <<"readme">>},
+                {source_url, ~B"https://github.com/eh/eh"},
+                {extras, [~B"README.md", ~B"LICENSE"]},
+                {main, ~B"readme"},
```

Comment:
Same here, maybe we could do `README.md` instead of `readme`?

Reply:
@paulo-ferraz-oliveira We can not, and we also need to discuss soft deprecating this feature. If we say `README`, then ex_doc will try to point to `README.html`, however, the `README.md` after conversion is lower cased. We added the support for lowering cases, so we can do that now, but if we are to deprecate it, then yeah...